docs/api.rst

```diff
@@ -2,43 +2,55 @@ API Documentation
 =================
 
 
-Messages
+TChannel
 --------
 
-.. automodule:: tchannel.messages
+.. automodule:: tchannel.tornado.tchannel
```

Comment:
We expect users to import just `tchannel.tornado` here. That re-exports pars of the public API.